.changeset/fresh-pets-eat.md

```diff
@@ -0,0 +1,6 @@
+---
+'@orchestrator-ui/orchestrator-ui-components': patch
+'wfo-ui': patch
```

Comment:
WFO UI does not need a version bump.

Reply:
Removed it.

I think it would be nice to have increasing version numbers (even when we don't publish a package to NPM)

That can also be done; when releasing it from main